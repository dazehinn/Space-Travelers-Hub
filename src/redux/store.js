import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './rockets/rocketsSlice';
import reducer from './missions/missionSlice';

const store = configureStore({
  reducer: {
    missions: reducer,
    rocket: rocketReducer,
  },
});

export default store;
