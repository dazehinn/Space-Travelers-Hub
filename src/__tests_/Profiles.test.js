import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Profiles from '../components/profiles';
import store from '../redux/Store';

it('Test Profiles component renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Profiles />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});