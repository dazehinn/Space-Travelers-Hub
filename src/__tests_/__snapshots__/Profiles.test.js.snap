// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Profiles component renders correctly 1`] = `
<div
  className="profileB"
>
  <div
    className="rocketProfile"
  >
    <h2>
      My Missions
    </h2>
    <p>
      Mission not joined yet
    </p>
  </div>
  <div
    className="rocketProfile"
  >
    <h2>
      My Rockets
    </h2>
    <p>
      No reserved rocket
    </p>
  </div>
</div>
`;
